docs: chroma and local db example (#7695)
langflow-ai/langflow: docs/docs/Components/components-vector-stores.md

|----------------|------------------------|--------------------------------|
| vector_store   | CouchbaseVectorStore    | A Couchbase vector store instance configured with the specified parameters. |

## Local DB

The **Local DB** component is Langflow's enhanced version of Chroma DB.

> It  that adds a user-friendly interface with two modes (Ingest and Retrieve), automatic collection management, and built-in persistence in Langflow's cache directory.

Author: ```suggestion
The component adds a user-friendly interface with two modes (Ingest and Retrieve), automatic collection management, and built-in persistence in Langflow's cache directory.
```
